### DISCUSSION TITLE
VIDEO TUTORIAL

### DISCUSSION THREAD

**Discussion by ghost** [aiogram/aiogram#924] [Ideas]
Hello! I wanted to suggest the idea of creating an aiogram bot video tutorial series or starting a YouTube channel focused on aiogram. It seems that aiogram simplifies the process of building bots and saves a lot of time. However, some newcomers might struggle to grasp the concepts and could benefit from more detailed guidance. While there are a few videos available on YouTube, having comprehensive tutorials that cover each feature in code would be highly valuable. I'm eager to learn and would greatly appreciate it if you could consider creating such resources. Thank you in advance!

**Comment by nkorgik** [comment]
Hi, did you find some good materials about creating the bot with _**Aiogram**_ framework? It would be great if there's some kind of instructions how to use **_Asyncio_** properly.
